### PR TITLE
fixed password strength indicator

### PR DESCRIPTION
**Pull request by stevetayloruk**
Problem: When special chars were used that were not in the regex it would show an amber bar and still let you create the site.

Solution: Added in more special chars and removed console.log statements

### CODE REVIEW

File: src/Orchard.Cms.Web/Modules/Orchard.Setup/Assets/js/strength.js [hunk 1]
```diff
@@ -21,7 +21,7 @@
         var upperCase = new RegExp('[A-Z]');
         var lowerCase = new RegExp('[a-z]');
         var number = new RegExp('[0-9]');
-        var specialchar = new RegExp('([!,%,&,@,#,$,^,*,?,_,~])');
+        var specialchar = new RegExp('([!,%,&,@,#,$,^,*,?,_,~,",�,(,)])');
```
Comment (sebastienros) [inline src/Orchard.Cms.Web/Modules/Orchard.Setup/Assets/js/strength.js]: Did you get this extended list from ASP.NET identity source code?

Comment (stevetayloruk) [inline src/Orchard.Cms.Web/Modules/Orchard.Setup/Assets/js/strength.js]: no, from my keyboard. ;-)

Comment (stevetayloruk) [inline src/Orchard.Cms.Web/Modules/Orchard.Setup/Assets/js/strength.js]: seriously though, it does pose the question which chars to support and which ones not to.

Comment (sebastienros) [inline src/Orchard.Cms.Web/Modules/Orchard.Setup/Assets/js/strength.js]: Just checked, and the aspnet identity validation option `RequireNonAlphanumeric` which means we just need to check for non alpha-numeric, i.e. `new RegExp('(~[A-Za-z0-9])')`

Comment (stevetayloruk) [inline src/Orchard.Cms.Web/Modules/Orchard.Setup/Assets/js/strength.js]: Ok, I'll update it

Comment (stevetayloruk) [inline src/Orchard.Cms.Web/Modules/Orchard.Setup/Assets/js/strength.js]: try now.

**Pull request merged**